src/actions/WalletListMenuActions.tsx

```diff
@@ -280,6 +281,21 @@ export function walletListMenuAction(
       }
     }
 
+    case 'goToParent': {
+      return async (dispatch, getState) => {
+        const state = getState()
+        const { account } = state.core
+        const { currencyWallets } = account
+        const wallet = currencyWallets[walletId]
+
+        navigation.navigate('transactionList', {
+          walletId,
+          tokenId: null,
+          walletName: wallet.name ?? wallet.currencyInfo.displayName
```

Comment:
Never access `wallet.name` directly. We have a helper function for this, `getWalletName`, that handles all the fallback stuff.